AsyncPoco.Tests/AsyncPoco.Tests.csproj

```diff
@@ -80,19 +85,20 @@
     <Compile Include="MultiPocoTests.cs" />
     <Compile Include="Misc.cs" />
     <Compile Include="ColumnMapper.cs" />
-    <Compile Include="PetaTest.cs" />
```

Comment:
PetaTest.cs is a carry-over from PetaPoco and is no longer needed now that this is relying on a more standardized testing framework. It was a very clever collection of code, but NUnit can take over those duties so that we don't have to. :)